src/axom/quest/examples/CMakeLists.txt

```diff
@@ -621,7 +621,7 @@ if(AXOM_ENABLE_TESTS AND AXOM_DATA_DIR)
     endif()
 endif()
 
-if (ENABLE_FORTRAN)
+if (ENABLE_FORTRAN AND FALSE)
```

Comment:
I had to disable these Fortran tests (quest's signed distance and inout Fortran tests) due to a linking issue in a `clang+ifort` configuration.

Should I revert this change and locally patch in my application, or leave it as is and add an issue to address this in the future. (I'll take responsibility for fixing it).

Reply:
It would be ideal to fix before merging.

Reply:
I reverted the change -- I'll try to handle it separately or add a local patch for my config.